site/en/docs/extensions/mv3/external_extensions/index.md

```diff
@@ -178,102 +148,156 @@ files. To see if this is the problem, follow these steps:
     external extension was not installed, repeat these steps until you do not see an error in the
     Console application.
 
+{% endDetails %}
+
+### Linux {: #preference-linux }
+
+1. Create a JSON file with the name of the extension ID. For example: `aaabbbcccdddeeefff.json`.
+1. Place it in one of the folders listed below:
+    - `/opt/google/chrome/extensions/`
+    - `/usr/share/google-chrome/extensions/`
```

Comment:
@rdcronin Are these paths still accurate? If we support multiple paths, when should developers prefer one over the other?

Reply:
I think this depends on Linux-y things (/usr/share and /opt are different folders with different semantics, I think?  This is fairly related [1], though talks about usr/local vs usr/share)  I don't know Linux management well enough to speak to it, and I'd also shy away from documenting it here.

[1] https://unix.stackexchange.com/questions/11544/what-is-the-difference-between-opt-and-usr-local

Reply:
I tried out `/opt/google/chrome/extensions/` and it works 👍